src/replay/components/player/VideoStreamer/ShakaVideoStreamer/shakaBitrateManager.js

```diff
@@ -35,6 +36,16 @@ function isUnique(item, index, arr) {
   return arr.indexOf(item) === index;
 }
 
+function manualbitrateSwitch(shakaPlayer: ShakaPlayer, configuration, track: ShakaTrack) {
```

Comment:
Please use a capital B for Bitrate in this function name.

Reply:
Resolved this function name as suggested.